ted2zim/entrypoint.py

```diff
@@ -75,19 +74,17 @@ def main():
 
     parser.add_argument(
         "--title",
-        help="Custom title for your project and ZIM. Default to Channel name (of first video if playlists)",
-        required=True,
+        help="Custom title for your project and ZIM. Default value - TED Collection",
+        default="TED Collection",
```

Comment:
we want something a bit smarter than that. See #33. We shall have different default whether it's a single topic or not. Kinda what we do for youtube.

Reply:
Done. Now we update title if not provided and consider number of topics that have valid videos.